src/lando/api/tests/test_diff_warnings.py

```diff
@@ -6,84 +8,93 @@
     DiffWarningStatus,
 )
 
-pytest.skip(allow_module_level=True)
-# NOTE: this should be ported as an API endpoint.
-
 
 @pytest.fixture
 def phab_header(phabdouble):
     user = phabdouble.user(username="test")
-    return {"X-Phabricator-API-Key": user["apiKey"]}
+    return {"HTTP_X_Phabricator_API_Key": user["apiKey"]}
 
 
 @pytest.fixture
 def diff_warning_data():
-    return {"message": "this is a test warning"}
+    return json.dumps({"message": "this is a test warning"})
 
 
-def test_diff_warning_create_bad_request(db, client):
+@pytest.mark.django_db(transaction=True)
+def test_diff_warning_create_bad_request(client):
     """Ensure a request that is missing required data returns an error."""
     response = client.post(
-        "/diff_warnings",
-        json={},
+        "/api/diff_warnings/",
+        data={},
+        content_type="application/json",
     )
     assert response.status_code == 400
 
 
-def test_diff_warning_create_bad_request_no_message(db, client, phab_header):
+@pytest.mark.django_db(transaction=True)
+def test_diff_warning_create_bad_request_no_message(client, phab_header):
     """Ensure a request with incorrect data returns an error."""
     response = client.post(
-        "/diff_warnings",
-        json={"revision_id": 1, "diff_id": 1, "group": "LINT", "data": {}},
-        headers=phab_header,
+        "/api/diff_warnings/",
+        data={"revision_id": 1, "diff_id": 1, "group": "LINT", "data": {}},
+        content_type="application/json",
+        **phab_header,
```

Comment:
Nit: according to https://docs.djangoproject.com/en/5.0/topics/testing/tools/#making-requests, 

> Keyword arguments starting with a `HTTP_` prefix are set as headers, but the `headers` parameter should be preferred for readability.

Both approaches work, so 🤷🏼‍♂️